apiserver/index.php

```
@@ -88,15 +88,15 @@ public function __construct() {
       $contents = file_get_contents($this->stateFile);
 
       if ($contents === FALSE) {
-        throw new \RuntimeException(sprintf('Failed to read data from the server state file %s', $this->stateFile));
+        throw new \RuntimeException(sprintf('Failed to read data from the server state file %s', $this->stateFile), 500);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '91', column '19'). (undefined)

(MissingImport)